addons/l10n_gcc_invoice/views/report_invoice.xml

```diff
@@ -280,17 +280,18 @@
                                         <span t-field="line.quantity"/>
                                         <span t-field="line.product_uom_id" groups="uom.group_uom"/>
                                     </td>
-                                    <td name="account_invoice_line_name" class="text-end">
-                                        <t t-set="line_name" t-value="
-                                            line.with_context(lang=o.partner_id.lang).product_id.display_name
-                                                if line.product_id and line.name in (
-                                                line.with_context(lang='ar_001').product_id.display_name,
-                                                line.with_context(lang='en_US').product_id.display_name
-                                            ) else line.name
-                                        "/>
-                                        <span t-out="line_name" t-options="{'widget': 'text'}"/>
-                                    </td>
-
+                                    <t t-if="o.partner_id.lang == 'ar_001'">
+                                        <td name="account_invoice_line_name" style="'text-align: right;'">
```

Comment:
What's the difference between `style="'text-align: right;'"` and `class="text-end"`?